### PR TITLE
feature-7

### PR DESCRIPTION
**Pull request by samuskitchen**
## Summary
- Added javadoc documentation
## Trello Card
https://trello.com/c/nA5IDu4u/47-javadoc

### CODE REVIEW

File: src/main/java/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -33,28 +33,58 @@ public BookController(BookRepository bookRepository) {
         return bookRepository.findAll();
     }
 
+    /**
+     * This method consults book by author
+     *
+     * @param bookAuthor
+     * @return the book object
```
Comment (Jimenarosello) [inline src/main/java/wolox/training/controllers/BookController.java]: Just as a suggestion, you can use {@link Book} to link the book class in the comment 😄

**Review by Jimenarosello — APPROVED** [review]
Nice job! Check out the `{@link Book}` for future reference 😊  🚀

File: src/main/java/wolox/training/controllers/BookController.java (hunk 1 above)
Comment (samuskitchen) [reply src/main/java/wolox/training/controllers/BookController.java]: changos, you're right

File: src/main/java/wolox/training/controllers/BookController.java [hunk 2]
```diff
@@ -33,28 +33,58 @@ public BookController(BookRepository bookRepository) {
         return bookRepository.findAll();
     }
 
+    /**
+     * This method consults book by author
+     *
+     * @param bookAuthor
```
Comment (JaimLus) [inline src/main/java/wolox/training/controllers/BookController.java]: I think adding a short description of the parameter would make the javadoc more complete.

**Review by JaimLus — APPROVED** [review]
Nice job!

**Pull request merged**